### ISSUE TITLE
The screenshot shortcut doesn't work 

### ISSUE THREAD

**Issue by Oluf775** [killergerbah/asbplayer#522]
Hello, when i press ctrl shift s for exporting a screenshot to the latest anki card, it copies the current subtitles instead of taking a screenshot.
Also when I press ctrl shif x to record and then a second time to stop the recording, it starts a new one during the first one.
Maybe I've missed something obvious, but when I use the app with animelon it works fine. With Asbplayer though, it seems to not follow the same rules

Thank you

**Comment by killergerbah** [comment]
Is this happening on streaming video or local video?

**Comment by Oluf775** [comment]
On the local video with asb player.

**Comment by killergerbah** [comment]
Yes you are right the rules on the local video player are different from streaming video.
However it would be great to get more details on what you want and what you expect since I am having trouble imagining what you are actually doing that looks buggy. On the local video player, the video will pause when you press Ctrl+Shift+X, so there's no opportunity to 'stop' a recording.

**Comment by killergerbah** [comment]
Manual "recording" behavior is now similar between app and extension.
